app/rasterize_app.cc

```diff
@@ -5,17 +5,17 @@
 
 int main(int argc, char** argv)
 {
-  if (argc != 2) {
-    std::cerr << "Usage: ./rasterize_app <path-to-gdb-directory>" << std::endl;
-    return 1;
-  }
+    if (argc != 2) {
```

Comment:
You should refrain from committing whitespace changes to a repository, as they disturbe the readability of the Git history. E.g., `git blame` would claim that you are the last author of this line after this change. I am aware that your IDE has done this automatically though.